test/coverlet.core.tests/Instrumentation/InstrumenterTests.cs

```diff
@@ -56,9 +74,26 @@ public void TestInstrument()
         }
 
         [Fact]
-        public void TestInstrumentCoreLib()
+        public void TestInstrumentWithExcludedFiles()
```

Comment:
I would remove this test...at the moment there is a bug in implementation...this PR should fix https://github.com/tonerdo/coverlet/pull/524
After that we won't need anymore to search file on parent and we can simply pass glob pattern.

Reply:
Done